packages/docs/docs/tutorials/data-modeling/family-relationships.md

```diff
@@ -0,0 +1,278 @@
+- However, if family members participate in clinical or billing activities, you can reference their participation by creating a [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource for them (see below).
+- Some examples of how a family member may participate in clinical / billing activities:
+  - Child/dependent is the subject of a clinical office visit, parents / family are in attendance
+  - Family member is the insurance subscriber / payor, patient is a spouse or dependent
+  - A Newborn (patient) is administered a medication by a parent
+
+1.  How often will Patients share family members?
+
+    - Depending on your practice setting, you may have mulitple patients who share family members.
+    - For example, two siblings might be enrolled as separate patients in a pediatric clinic, but they share the same mother and father.
+    - In these situations, it is desireable to store demographic and contact information in single resource, to make it easier to update.
+    - For example, you would want a single resource representing both siblings' father, to make sure his information was always up to date.
+    - However, because a [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource models the relationship between a person and a patient, each sibling would need it's own [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource linking them to the father
+    - Therefore, you will need to model the father as either a[Person](/docs/api/fhir/resources/person) or a Patient, and use either the Patient/[RelatedPerson](/docs/api/fhir/resources/relatedperson)/Person or Patient/[RelatedPerson](/docs/api/fhir/resources/relatedperson)/Patient schemes below to model the family.
+
+2.  Which family members will get need clinical information (test results, clinical notes, lab specimens), if any?
+
+    - Next, we should try to estimate which family members will be the "focus" of clinical activities and will be the subject of clinical information.
+    - Some examples of clinical activities are:
+      - Office / Telehealth visit with a physician
+      - Receipt of a medication or prescription
+      - Performance of imaging or labwork
+      - Receipt of a clinical procedure or operation
+    - Each individual who is the focus of a clinical activity should be modeled as a[Person](/docs/api/fhir/resources/patient)resource.
+    - For example:
+      - In pediatric virtual therapy, a single child may be the focus, even if parents are attending sessions
+      - In a maternal care situation, some medications and treatments may be prescribed for the mother, who is the focus of those activities. Additonal therapies might be prescribed for the child, who would be the focus of those activities.
+
+3.  Is the subject of the treatment an indivdual patient, or a family unit?
```

Comment:
Make specific note of Training and Education for family members as this is common for this use case